### PR TITLE
[Sample Data: Bulk Uploads] Move numeric_ids under more autograding examples

### PR DESCRIPTION
**Pull request by shailpatels**
### Please check if the PR fulfills these requirements:

* [x] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [x] Tests for the changes have been added/updated (if possible)
* [x] Documentation has been updated/added if relevant

### What is the new behavior?
Moves numeric_ids.txt to more_autograding_examples/bulk_uploads/submissions
and adds a readme description explaining what those files are

### CODE REVIEW

**Comment by MasterOdin** [comment]
Could the user ids be listed in `04_numeric_ids.txt`? Is that file read-in "as is" somewhere that requires it to be formatted the way it is, or could adding more data to the header/footer of that txt file be fine?

**Comment by shailpatels** [comment]
Yeah that file isn't being read, made changes now

File: .setup/data/users.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@
 #   user_id: string representing the id for the user both in the database as well as linux
 #
 #   user_numeric_id: string represented the numeric id, e.g a RIN used in the database
+#         by default this is null in the database, adding this value will populate the numeric_id column 
+#         for the sample user being created, see more_autograding_examples/bulk_upload_pdfs/submissions/numeric_ids.txt for a list of ids to use
```
Comment (MasterOdin) [inline .setup/data/users.yml]: update the name and put the stuff after it on a newline:
```
#   user_numeric_id: string represented the numeric id, e.g a RIN used in the database
#         by default this is null in the database, adding this value will populate the numeric_id column 
#         for the sample user being created, see more_autograding_examples/bulk_upload_pdfs/submissions/04_numeric_ids.txt 
#         for a list of ids to use
```

File: more_autograding_examples/bulk_upload_pdfs/submissions/readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+* qr_six_page_ contains 36 pages with a QR code every 6 pages
+* qr_prefix_ is the same length with the prefix 'sample_data' encoded on every QR
+* qr_stress_test_ contains QR code placed in varying lengths and has a prefix and suffix that contains URL components
+
+### 04_numeric_ids.txt
+
+04_numeric_ids.txt are a list of fake numeric_ids that are used in setting up sample users,
+the associated user id is listed next to the numeric id but not all are being used currently
+
+These fake numeric ids are being used in the Submitty bulk upload OCR scanner.
```
Comment (MasterOdin) [inline more_autograding_examples/bulk_upload_pdfs/submissions/readme.md]: I think this is clearer:
```
04_numeric_ids.txt is a list of fake numeric_ids that are shown in 04_numeric_id_scan.pdf
and is used to test the bulk upload OCR scanner feature. Some of the ids in this txt file are
associated with the user_id of one of the fake generated test users. When adding a
new fake user yml, please use one of the ids in this file for their numeric_id, and put the
user_id next to the numeric_id in the txt file.
```

The file is not used directly used in setting up sample users, but just as a reference point for making the user yml files. Let's not insinuate something more.

**Comment by shailpatels** [comment]
No problem, if it makes things easier to understand I'm happy to make the changes!

**Pull request merged**